sources/api/netdog/src/cli/generate_hostname.rs

```diff
@@ -0,0 +1,32 @@
+use super::{error, print_json, Result};
+use crate::CURRENT_IP;
```

Comment:
What do you think about moving the constants into some other module, like the parent?

Reply:
I spent more time than I'd like to admit playing and thinking about that...  I ended up keeping them where they were since they get used in the CLI _and_ other modules and I didn't really like the idea of making a `lib.rs` just for constants.

Totally willing to entertain this though - perhaps there's an elegant way I just couldn't come up with! :smile: